.gitbook/overview/roadmap.md

```diff
@@ -1,63 +1,37 @@
 
-### 2022 Q3: Data Exchange Protocol v0 on Testnet
+---
 
-- Evaluate data exchange protocol v0 on Panacea Testnet
-  - Technical evaluation
-  - Product-wise evaluation
-- Update and release new version of whitepaper
+In order to resolve these challenges, the team is developing this data market protocol with several latest technologies. 
 
-### 2022 Q4: Data Marketplace Service
+### Data quality validation in Secure Enclaves
 
-- Release data exchange protocol v0 on Panacea Mainnet
-- Release data marketplace service based on data exchange protocol
-  - Early corporate participants joined
+Data provided by sellers should be validated according to criteria that buyers specified. This validation process can be performed on Panacea because Panacea is already a Byzantine fault-tolerant consensus system. However, we should not put sensitive data on any public blockchain even if the data is encrypted. So, we introduced a new role: Verified Off-chain Data Validators with Secure Enclaves. Secure enclave technologies such as Intel SGX empowers data validators to validate data without exposing the content of data outside of the enclave. It means that operators of the data validator do their job without looking into the content of data. 
 
-### 2023 Q1~2: Carving Data Exchange Protocol
+### W3C Verifiable Credentials and Zero-knowledge Proof
 
-- Define/develop data exchange protocol v1
-  - Improved data quality validation and privacy
-    - W3C Verifiable Credentials
-    - Zero-knowledge Proof
-  - Improved usability and tokenomics
-  - Enhanced security
-  - DAO
-  - Cross-chain NFT (compatible with NFT marketplaces)
+The protocol should support various ways for data buyers to specify what types of data they want to buy. We believe that W3C Verifiable Credential may be one of the solutions. Based on this standard, selective disclosure can also be achieved by zero-knowledge proof. 
 
-### 2023 Q3: Healthcare Services on Data Exchange Protocol
+In addition to these technologies above, the data market protocol is going to provide rich feature sets for service providers who build services on the top of the data market protocol in various legal countries and industries. It will be helpful for them to build their service in compliance with their law for handling personal data. For example, the protocol is going to provide features for making consents to data sharing/exchange signed cryptographically.
```

Comment:
```suggestion
In addition to these technologies above, the data exchange protocol is going to provide rich feature sets for service providers who build services on the top of the data exchange protocol in various legal countries and industries. It will be helpful for them to build their service in compliance with their law for handling personal data. For example, the protocol is going to provide features for making consents to data sharing/exchange signed cryptographically.
```